app/mailers/agency_update_mailer.rb

```diff
@@ -0,0 +1,16 @@
+class AgencyUpdateMailer < ApplicationMailer
+
+  def new_agency_update_email
+    @agency = params[:agency]
+    @aur = params[:agency_update_request]
+
+    mail(from: @agency.email, to: 'admin@saf-unite.org', cc: (Proc.new { User.where(role: 'admin').pluck(:email) }).call, subject: 'You got a new organization update request!')
```

Comment:
There are few things in this line

1. The "from" should not be the agency's email because that email is not the one sending the email. We are sending the email so for now put `noreply@conectatecarolina.org`
2. Instead of "from" we should add `reply_to` so that saf can click on reply and send them an email if they need to.
2. You don't need a Proc here you can just use `User.where(role: 'admin').pluck(:email)`
3. We need to ask if we want to always send emails to `admin@saf-unite.org` right now as it is, saf can't configure that, we would need to deploy the app again to change it, which is not good.